Pony/source/Address.cpp

```diff
@@ -27,6 +27,26 @@ unsigned Address::GetAddress() const
     return m_address;
 }
 
+unsigned char Address::GetFst() const
```

Comment:
Could be a function with a position argument, because this code already presumes that the user code knows what it wants.

Reply:
```cpp
uint8_t Address::GetAddressSegment(uint8_t position)
{
    return (m_address >> (32 - 8 * position));
}
``` 
?